src/AzureExtension/Widgets/Templates/AzureLoadingTemplate.json

```diff
@@ -7,11 +7,11 @@
             "type": "Container",
             "items": [
                 {
-                    "type": "TextBlock",
-                    "text": "%Widget_Template/Loading%",
-                    "wrap": true,
-                    "weight": "bolder",
-                    "horizontalAlignment": "Center"
+                  "type": "TextBlock",
+                  "text": "${if(loadingMessage, loadingMessage, '%Widget_Template/Loading%')}",
+                  "wrap": true,
+                  "weight": "bolder",
+                  "horizontalAlignment": "Center"

```

Comment:
We seem to be using 4 spaces in the rest of the file

Reply:
Fixed